packages/google_maps_flutter/google_maps_flutter/example/lib/place_polygon.dart

```diff
@@ -96,6 +96,8 @@ class PlacePolygonBodyState extends State<PlacePolygonBody> {
 
     setState(() {
       polygons[polygonId] = polygon;
+      polygonOffsets[polygonId] = _polygonIdCounter.ceilToDouble();
+      _polygonIdCounter++;
```

Comment:
Can we add comments on what does this code do?

Reply:
done